health/README.md

```diff
@@ -0,0 +1,29 @@
+# Health Package
+
+Package contains tools that allow to add REST endpoint for handling `health` and `readiness` requests.
+Package also provides implementation for most typical checks that may be done to figure our if service is healthy/ready.
+
+## REST Handler
```

Comment:
It's not really a REST API, so much as just a plain HTTP Endpoint that serves one specific purpose. IMO, I think you can just say _HTTP Handler_ or _Probe Endpoints_. If you're looking for synonyms, maybe check out the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).